hepcrawl/spiders/dnb_spider.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015 CERN.
```

Comment:
All these new files should have 2016 copyright year. For existing files that was modified you add 2016 to the list.